app/Http/Controllers/DashboardController.php

```diff
@@ -20,11 +20,11 @@ public function index() : View
     {
         $awaitingFeedback = auth()->user()->feedback()->with(['taskDelegation', 'project.task.course'])->unreviewed()->get()
             ->filter(fn(ProjectFeedback $feedback) => $feedback->taskDelegation->deadline_at->isFuture());
-        $courses = auth()->user()->courses()->get();
+        $courses = auth()->user()->courses()->orderBy('created_at', 'asc')->get();
```

Comment:
Skal det ikke være desc? Så det nyeste kursus altid står øverst

Reply:
remember to update the tests too